tests/console/shells.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test all officially SLE supported shells
+# Maintainer: Oliver Kurz <okurz@suse.de>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+
+sub run() {
+    select_console 'user-console';
+    zypper_call 'in ksh tcsh zsh';
+    assert_script_run 'ksh -c "print hello" | grep hello';
+    assert_script_run 'sh -c "echo hello" | grep hello';
```

Comment:
not sure what kind of test this is. This is *really* 10.000 miles, but as long as its in extra tests I don't mind